cypress/support/commands.js

```diff
@@ -137,3 +158,26 @@ Cypress.Commands.add('visitWithoutFetch', path => {
         }
     );
 });
+
+Cypress.Commands.add(
+        'accessibilityCheck',
+    (context = null, options = null) => {
+        cy.injectAxe();
+        cy.checkA11y(context, options, terminalLog);
+    }
+);
+
+// This command opens the page with accessibility checking.
+// path: page's path.
+// context (optional):
+// Defines the scope of the analysis - the part of the DOM that you would like to analyze.
+// This will typically be the document or a specific selector such as class name, ID, selector, etc.
+// options (optional):
+// Set of options passed into rules or checks, temporarily modifying them.
+Cypress.Commands.add(
+    'visitWitAccessibilityCheck',
```

Comment:
```suggestion
    'visitWithAccessibilityCheck',
```

Apologies as I may have missed this before, but this may be a typo